tests/stp/pubsub/pubsub.csv

```diff
@@ -37,4 +37,10 @@ run,0,0,q,psHandle(upsert;`trade;testtrade),1,,"Update trade table on pubsub"
 true,0,0,q,1~count psHandle(`trade),1,,"Check there is one row in trade table"
 run,0,0,q,neg[psHandle](`.stpps.pubclear;`trade),1,,"Call pubclear funtion"
 true,0,0,q,0~count psHandle(`trade),1,,"Check that trade table has been cleared"
-true,0,0,q,1~.tst.upd,1,,"Check that local UPD function has been called"
\ No newline at end of file
+true,0,0,q,1~.tst.upd,1,,"Check that local UPD function has been called"
+run,0,0,q,upd:{show x:show count y},1,,"Define upd for sub filter tests"
+run,0,0,q,rdbHandle(`.u.sub;`trade;`AAPL`GOOG),1,,"Subscribe to filtered sym list"
+run,0,0,q,rdbHandle".stpps.subrequestfiltered",1,,"Check filtered subscription has passed"
+run,0,0,q,rdbHandle".u.pub[`trade;-10 sublist trade]",1,,"Force publish for the filtered list"
```

Comment:
I think you need to add another test to check that this publish worked as expected - currently these tests just make sure that the `.u.pub` function will run, but don't check that it does what it's supposed to